### ISSUE TITLE
Option for using http over https

### ISSUE THREAD

**Issue by icbat** [nodkz/mongodb-memory-server#172]
We're using this on a network where SSL verification to the outside never succeeds, and our usual (bad) approach is to turn off strict SSL. For instance, our .yarnrc has `strict-ssl false`.

Is there any way to easily configure that here? A glance of the code leads me to think there's no option to send now.

Thanks!

Error logs from install:

```
error /<redacted path>/node_modules/mongodb-memory-server: Command failed.
Exit code: 1
Command: node ./postinstall.js
Arguments:
Directory: /<redacted path>/node_modules/mongodb-memory-server
Output:
mongodb-memory-server: checking MongoDB binaries cache...
events.js:167
      throw er; // Unhandled 'error' event
      ^

Error: unable to verify the first certificate
    at TLSSocket.onConnectSecure (_tls_wrap.js:1049:34)
    at TLSSocket.emit (events.js:182:13)
    at TLSSocket._finishInit (_tls_wrap.js:631:8)
Emitted 'error' event at:
    at TLSSocket.socketErrorListener (_http_client.js:392:9)
    at TLSSocket.emit (events.js:182:13)
    at emitErrorNT (internal/streams/destroy.js:82:8)
    at emitErrorAndCloseNT (internal/streams/destroy.js:50:3)
    at process._tickCallback (internal/process/next_tick.js:63:19)
```

**Comment by icbat** [comment]
Oof, it's not just a problem download, this also occurs if you use the `-core` library with mongo installed and running, and try to use it.

**Comment by hasezoey** [comment]
currently (still) there is no option to disable HTTPS / SSL

**Comment by hasezoey** [comment]
im currently looking into this, it looks like ssl can already be disabled with environment variable `npm_config_strict_ssl` (this variable gets set by npm or yarn [tested both])

but there is currently no way to change the protocoal (https to http)

**Comment by hasezoey** [comment]
closing because included since `7.0.0-beta.23`
